src/migrations/1690556226140_multiple_vestings.ts

```diff
@@ -0,0 +1,9 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+
+import Model from '../entities/Proposal/model'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.alterColumn(Model.tableName, 'vesting_address', { type: 'TEXT[]', using: "CASE WHEN vesting_address IS NULL THEN '{}'::TEXT[] ELSE ARRAY[vesting_address] END"})
```

Comment:
Shouldn't this be `WHEN vesting_address IS NULL THEN ARRAY[]` ?

I wonder what happens if you put an empty array as text and then try to save an array with elements.

Reply:
This is why the cast is needed
![image](https://github.com/decentraland/governance/assets/45410089/92516db8-ffeb-44b1-9f26-61f70e820b7e)